deps/rabbit/include/rabbit_khepri.hrl

```diff
@@ -0,0 +1,63 @@
+%% This Source Code Form is subject to the terms of the Mozilla Public
+%% License, v. 2.0. If a copy of the MPL was not distributed with this
+%% file, You can obtain one at https://mozilla.org/MPL/2.0/.
+%%
+%% Copyright (c) 2024 Broadcom. All Rights Reserved. The term “Broadcom”
+%% refers to Broadcom Inc. and/or its subsidiaries. All rights reserved.
+%%
+
+%% This header has macros that define the `khepri_path:native_pattern()'
+%% path patterns used for each piece of metadata in the store. We use macros
+%% for these so that we can pattern match on these path patterns as well as
+%% create them as new terms.
+%%
+%% If you are creating a path pattern to use in a call to the Khepri API (for
+%% example `rabbit_khepri:get/1') you should prefer using the
+%% `khepri_<entity>_path' function from the `rabbit_db_<entity>' modules
+%% instead, for example `rabbit_db_queue:khepri_queue_path/2', since those
+%% functions have guards to assert that the variables passed are valid pattern
+%% components.
+
+-define(KHEPRI_ROOT_PATH, ?KHEPRI_ROOT_PATH([])).
+-define(KHEPRI_ROOT_PATH(Rest), [rabbitmq | Rest]).
```

Comment:
Now that I think of it, we should not use the `KHEPRI_` prefix here. Because there is a risk that it pollutes Khepri's namespace. Could please rename them to use another prefix such as `RABBITMQ_KHEPRI_`?

Reply:
I mean all macros, not just the two I quoted.